File: java/code program/Mobil.java
public class Mobil extends Kendaraan {
    private int doors;
    private String fuelType;

    public Mobil(String brand, String model, int year, double price, int doors, String fuelType) {
        super(brand, model, year, price);
        this.doors = doors;
        this.fuelType = fuelType;
    }

    public int getDoors() {
        return doors;
    }

    public void setDoors(int doors) {
        this.doors = doors;
    }

    public String getFuelType() {
        return fuelType;
    }

    public void setFuelType(String fuelType) {
        this.fuelType = fuelType;
    }
}

File: python/code program/mobil.py
from kendaraan import Kendaraan


class Mobil(Kendaraan):
    def __init__(self, brand="", model="", year=0, price=0.0, doors=0, fuel_type=""):
        super().__init__(brand, model, year, price)
        self.doors = doors
        self.fuel_type = fuel_type

    def set_jumlah_pintu(self, doors):
        self.doors = doors

    def set_fuel_type(self, fuel_type):
        self.fuel_type = fuel_type

    def get_doors(self):
        return self.doors

    def get_fuel_type(self):
        return self.fuel_type
